### PR TITLE
Add the voltage level id of equipment in ElasticSearch

### PR DESCRIPTION
**Pull request by SlimaneAmar**
Signed-off-by: Slimane AMAR <slimane.amar@rte-france.com>

### CODE REVIEW

File: src/test/java/org/gridsuite/modification/server/EquipmentInfosServiceTests.java [hunk 1]
```diff
@@ -42,31 +41,33 @@
...
     }
 
     private void addEquipmentInfos(Set<String> ids, EquipmentInfos equipmentInfos) {
-        ids.add(equipmentInfosService.add(equipmentInfos).getEquipmentId());
+        ids.add(equipmentInfosService.add(equipmentInfos).getId());
     }
 
     @Test
-    public void testBadEquipmentType() {
+    public void testBadtype() {
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/modification/server/EquipmentInfosServiceTests.java:70]: testBadType

File: src/main/java/org/gridsuite/modification/server/dto/EquipmentInfos.java [hunk 2]
```diff
@@ -23,17 +30,60 @@
...
+            return Set.of(
+                hvdcLine.getConverterStation1().getTerminal().getVoltageLevel().getId(),
+                hvdcLine.getConverterStation2().getTerminal().getVoltageLevel().getId()
+            );
+        } else if (identifiable instanceof Branch) {
+            Branch<?> branch = (Branch<?>) identifiable;
+            String vlId1 = branch.getTerminal1().getVoltageLevel().getId();
+            String vlId2 = branch.getTerminal2().getVoltageLevel().getId();
+            return vlId1.equals(vlId2) ? // Internal line ?
+                Set.of(branch.getTerminal1().getVoltageLevel().getId()) :
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/modification/server/dto/EquipmentInfos.java:73]: Reuse vlId1

File: src/main/java/org/gridsuite/modification/server/dto/EquipmentInfos.java [hunk 3]
```diff
@@ -23,17 +30,60 @@
...
+                Set.of(
+                    branch.getTerminal1().getVoltageLevel().getId(),
+                    branch.getTerminal2().getVoltageLevel().getId()
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/modification/server/dto/EquipmentInfos.java:76]: Reuse vlId1 and vlId2

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/test/java/org/gridsuite/modification/server/EquipmentInfosServiceTests.java (hunk 1 above)
Comment (SlimaneAmar) [reply src/test/java/org/gridsuite/modification/server/EquipmentInfosServiceTests.java]: Done

**Pull request merged**